docs/developing-contracts/gas-reservation.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 7
+---
+
+# Gas reservation
+
+Gas reservation is the powerful feature of Gear Protocol that enables the new approach to smart-contract programming and modern [use cases](../gear/distinctive-features).
+
+Briefly, it allows sending messages using reserved gas from the initial message instead of using gas from the last sent message.
```

Comment:
Gas can be reserved not only in the initial message. I think it is better to say:
```suggestion
Briefly,  a program can send a message using gas that was reserved before instead of using gas from the currently processing message.
```

Reply:
ok